presto-spark-base/src/test/java/com/facebook/presto/spark/PrestoSparkQueryRunner.java

```diff
@@ -379,14 +379,13 @@ public PrestoSparkQueryRunner(
         }
         pluginManager.installPlugin(new HivePlugin("hive", Optional.of(metastore)));
 
-        Map<String, String> properties = ImmutableMap.<String, String>builder()
-                .put("hive.allow-drop-table", "true")
-                .put("hive.allow-rename-table", "true")
-                .put("hive.allow-rename-column", "true")
-                .put("hive.allow-add-column", "true")
-                .put("hive.allow-drop-column", "true")
-                .putAll(hiveProperties).build();
-        connectorManager.createConnection("hive", "hive", properties);
+        Map<String, String> properties = new HashMap<>(hiveProperties);
```

Comment:
@pdabre12 Is there a reason to not use the `builder()`?

Reply:
Why do we need this change?

Reply:
The reason I did not use the `builder()` is because I wanted to make use of the putIfAbsent() method available on HashMap and as far as I know we cannot create an HashMap with a `builder()`.

Reply:
This change is needed because under the hood while creating a queryRunner using `PrestoSparkNativeQueryRunnerUtils` , the `native worker properties (hiveProperties above)` are added to the properties. 

Not adding the `putIfAbsent()` check there throws an error : `duplicate key values for hive.allow-drop-table` .

Reply:
Can we simply remove `.put("hive.allow-drop-table", "true")` below to avoid the duplicate key issue?

Reply:
No we cannot because the `PrestoSparkNativeQueryRunnerUtils` uses the `PrestoSparkQueryRunner` present in `presto-spark-base` and `PrestoSparkQueryRunner` is called separately too while testing `presto-spark-base` operations. Removing these properties from here results in an error while running those tests.

Reply:
Then likely getNativeWorkerHiveProperties is not the right place to add this.
Can we add inside at presto-hive/src/test/java/com/facebook/presto/hive/HiveQueryRunner.java:225?